src/main/network/router/codel_queue.rs

```diff
@@ -406,32 +406,29 @@ mod tests {
         // Not above target so interval is not set.
         assert!(cdq.interval_end.is_none());
         let now = start + TARGET - one;
-        assert_eq!(cdq.process_standing_delay(&now, TARGET - one), false);
+        assert!(!cdq.process_standing_delay(&now, TARGET - one));
         assert!(cdq.interval_end.is_none());
```

Comment:
iirc I think this used the true/false for readability. (It's easier to see the difference between `assert_eq(long_string, true)` and `assert_eq(long_string, false)` compared to `assert(long_string)` and `assert(!long_string)`.) My opinion leans slightly towards the true/false, but it doesn't really matter to me.

Reply:
Ah right. I don't have a preference of one vs the other, so lean towards it not being worth fighting clippy/convention here